### DISCUSSION TITLE
Theme-ui dosen't install with react version ^17.0.2

### DISCUSSION THREAD

**Discussion by APiligrim** [system-ui/theme-ui#2364] [Q&A] [answered]
**Describe the bug**
Hello team at Theme-UI, 

I was trying to install theme-ui to an existing project and when I ran `npm install theme-ui @emotion/react` the error said that theme-ui 's version `0.15.4` is lower than my current react version `^17.0.2.` 

Would theme-ui's dependencies be updated anytime soon? Or are there any other way to fix this issue aside from downgrading my current version of react?

Thank you for your help!

**To Reproduce**
Install Theme-ui on a project with react version 17.0.2.

**Error Message**
```

! node_modules/react
npm ERR!   react@"^17.0.2" from the root project
npm ERR!   peer react@">=16.8.0" from @emotion/react@11.10.5
npm ERR!   node_modules/@emotion/react
npm ERR!     @emotion/react@"*" from the root project
npm ERR!     peer @emotion/react@"^11" from @theme-ui/core@0.15.4
npm ERR!     node_modules/@theme-ui/core
npm ERR!       @theme-ui/core@"*" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@">=18" from @theme-ui/core@0.15.4
npm ERR! node_modules/@theme-ui/core
npm ERR!   @theme-ui/core@"*" from the root project
```

**Comment by lachlanjc** [comment, marked as answer]
Hey there! [As of v0.15](https://github.com/system-ui/theme-ui/blob/develop/CHANGELOG.md#drop-support-for-react-16--17-2215), Theme UI only supports React 18+ (similar to other major frameworks, such as Next.js & Gatsby). Try upgrading your React, or if that's not possible, installing a prior version of Theme UI.

**Reply by APiligrim** [reply]
Thank you!
